beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -256,3 +256,17 @@ func emptyPayload() *enginev1.ExecutionPayload {
 		BlockHash:     make([]byte, fieldparams.RootLength),
 	}
 }
+
+func emptyPayloadCapella() *enginev1.ExecutionPayloadCapella {
```

Comment:
Why aren't transactions initialized? Same question for the already existing `empyPayload`.

Reply:
because it's empty. By definition transaction field is a list not vector